### ISSUE TITLE
Load local trailer over YouTube, if available

### ISSUE THREAD

**Issue by Scrub000** [jarnedemeulemeester/findroid#830]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by sannidhyaroy** [comment]
Yep, I've noticed this long time back too, that there's no way to play the local trailer on Findroid. It always redirects me to YouTube.
